### PR TITLE
Use XML-style closing tag for explicitly stated self-closing tags in HTML as well

### PR DESCRIPTION
**Pull request by TimothyGu**
Fixes jadejs/jade#2082.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -531,7 +531,7 @@ Compiler.prototype = {
       this.buffer('<');
       bufferName();
       this.visitAttributes(tag.attrs, tag.attributeBlocks.slice());
-      this.terse
+      this.terse && !tag.selfClosing
```
Comment (alubbe) [inline index.js]: For readability, could we use if/else instead of the conditional expression?

Comment (TimothyGu) [inline index.js]: Sure.

**Comment by ForbesLindesay** [comment]
Would it make more sense just to always include the `/`?  For performance, it's negligible and it would simplify things?

**Comment by TimothyGu** [comment]
@ForbesLindesay Do you mean that we should add a slash for regular HTML5 tags as well? I'd argue that the output is cleaner if we do not do that.

**Comment by TimothyGu** [comment]
@ForbesLindesay ping.

**Comment by ForbesLindesay** [comment]
Yes, that is what I meant, but I don't have a strong opinion on this.

**Pull request merged**